Unreal/CarlaUE4/Plugins/Carla/Source/Carla/Sensor/PixelReader.h

```diff
@@ -218,19 +219,25 @@ void FPixelReader::SendPixelsInRenderThread(TSensor &Sensor, bool use16BitFormat
                 }
                 #endif
 
-                // network
-                SCOPE_CYCLE_COUNTER(STAT_CarlaSensorStreamSend);
-                TRACE_CPUPROFILER_EVENT_SCOPE_STR("Stream Send");
-                Stream.Send(Sensor, BufView);
+                if (Sensor.AreClientsListening()) 
```

Comment:
Note that if is redundant because `FuncForSending` `PayLoad` is only called if `WritePixelsToBuffer` is called.

I suggest remove this if and keep the ode as it was.